allennlp/run.py

```diff
@@ -0,0 +1,128 @@
+from typing import Any, Dict, Union
```

Comment:
@schmmd and I were discussing this the other day. I was thinking potentially something analogous to the way `bin/aristo` works with subcommands. so we might have something like 

```
$ ./allennlp.py --help
$ ./allennlp.py train [...]
$ ./allennlp.py evaluate [...]
```

If I were doing that, I would probably create a `commands` directory with subdirectories for each of the commands and its helper functions.

thoughts about this approach?